### ISSUE TITLE
Socket hangs up when make post webhook auth request on windows

### ISSUE THREAD

**Issue by Stalso** [deepstreamIO/deepstream.io#895]
Hello. I have `deepstream` server (`3.1.2` version) installed as npm package on windows 8.1 or windows 10 (issue presents on both systems) developer machine. It uses simple `express` server as auth server for webhook auth. 

When I attempt to connect and login I am getting this output

    INCOMING_CONNECTION | from undefined (127.0.0.1)
    AUTH_ERROR | http auth error: Error: socket hang up
    INVALID_AUTH_DATA | invalid authentication data

All other stuff (RPC, events e.t.c) seems to work fine. Problem is in auth POST request. I should pay attentions that sometimes login works (but most of the time it fails). In fact, this issue is closely related to [this](https://github.com/deepstreamIO/deepstream.io/issues/816) . If I use `3.0.0` version of deepstream - everything works. If I use `3.1.0` as npm package - it fails, but if I use `3.1.0` as windows build from your release page, everything works fine (I do not know the reason). Also I have tested this on linux (`Ubuntu`) and everything works fine with all versions. 

To reproduce the bug, I have created three projects:

- [ds_auth_server](https://github.com/Stalso/ds_auth_server) - this is simple auth server
- [ds_server](https://github.com/Stalso/ds_server) - this is simple deepstreamIO server
- [ds-react-client](https://github.com/Stalso/ds-react-client) - this is simple react client

**Comment by yasserf** [comment]
Hello,

Thanks for raising. Any chance you have a reproducible repo I can use to test/fix this?

**Comment by Stalso** [comment]
@yasserf I provided you with three repos (react client, auth server and dsserver) in my original post

**Comment by yasserf** [comment]
Hey,

If anyone stumbles on this issue would be super grateful if you could try and see if this fails with V4, I don't have a windows setup available ATM and my mac is running into issues with running a VM for some reason 😞

**Comment by slachtar** [comment]
Hello,
I made tests under window 10 and here's the output on browser, webhook and deepstream server.

**chrome console**
```
Login result:  true
Login data:  {favouriteColour: "blue"}
```
**deepstream server console**
```
     _                     _
   __| | ___  ___ _ __  ___| |_ _ __ ___  __ _ _ __ ____
  / _` |/ _ \/ _ \ '_ \/ __| __| '__/ _ \/ _` | '_ ` _  \
 | (_| |  __/  __/ |_) \__ \ |_| | |  __/ (_| | | | | | |
  \__,_|\___|\___| .__/|___/\__|_|  \___|\__,_|_| |_| |_|
                 |_|
 =====================   starting   =====================
INFO | logger ready: std out/err
INFO | deepstream version: 3.2.5
INFO | configuration file loaded from ./config.yml
INFO | storage ready: no storage connector specified
INFO | cache ready: local cache
INFO | authenticationHandler ready: http webhook to http://127.0.0.1:3001/auth/check_socket_token
INFO | permissionHandler ready: valve permissions loaded from C:\Users\Hp\Desktop\test_ds\ds_server-master\ds_server-master\src\permissions.yml
INFO | Listening for websocket connections on 127.0.0.1:6021/deepstream
INFO | Listening for health checks on path /health-check
INFO | connectionEndpoint ready: µWebSocket Connection Endpoint
INFO | Listening for http connections on 0.0.0.0:8080
INFO | Listening for health checks on path /health-check
INFO | connectionEndpoint ready: HTTP connection endpoint
INFO | Deepstream started
INCOMING_CONNECTION | from  (127.0.0.1)
AUTH_SUCCESSFUL | OPEN
```
**web hook output**
```
Auth server running at localhost:3001
/auth/check_socket_token request:  {
  connectionData: { remoteAddress: '127.0.0.1', headers: {}, referer: '' },
  authData: { username: 'webagent', UUID: 'webagent', type: 'webhook' }
}
```
The most difficult part was starting react app :slightly_smiling_face: One have to run
```
npm install create-react-app
npm start
```

**Comment by yasserf** [comment]
Amazing thank you for testing! 

Closing this issue then as fixed in 5.0

Thank you @Stalso for the incredible bug report and apologies for the insane delay
